### PR TITLE
executable name can now be customized

### PR DESCRIPTION
**Pull request by ViviYe**
set a new variable f1-executable-name (currently set to sdaccel) instead of assuming the executable name is host and run ./host.

### CODE REVIEW

**Comment by sampsyo** [comment]
Seems cool! But maybe we should just have one `EXECUTABLE_NAME` option—there's no particular reason why the SDSoC executable is called `sdsoc`, for example. It could just be called `exe` by default. :smiley:

**Comment by ViviYe** [comment]
Should I just change the executable name to exe in both builedbot and in fuse-benchmarks? 😀

**Comment by sampsyo** [comment]
Hmm; I'm not sure what you mean by changing it in fuse-benchmarks… the way this works for SDSoC, at least, is that the configured `EXECUTABLE_NAME` here just overrides the default specified in the Makefile. So the Makefile can use whatever sensible name it wants! The Buildbot just overrides it with one that it knows how to invoke.

**Comment by ViviYe** [comment]
Thanks so much! That makes sense! Just being curious, how do the variables set in buildbot override variables in Makefile?

**Comment by sampsyo** [comment]
They work by invoking the `make` command with its special variable-setting argument syntax, like this:

```
make FOO=bar
```

That command invokes the `Makefile` in the current directory, while setting the variable `$FOO` to `bar`. Even if that file contains an assignment:

```
FOO := baz
```

Then the assignment will be ignored, and `$FOO` will never contain `baz`—only `bar`.

**Comment by ViviYe** [comment]
Thanks so much!!!

**Comment by ViviYe** [comment]
Just changed sdaccel also to use the same ```EXECUTABLE_NAME``` variable!

File: buildbot/buildbot/worker.py [hunk 1]
```diff
@@ -492,15 +492,20 @@ def stage_fpga_execute(db, config):
             # binary or the emulation executable.
             if task['mode'] == 'hw':
                 exe_cmd = ['sudo', 'sh', '-c',
-                           'source /opt/xilinx/xrt/setup.sh ; ./host']
+                           'source /opt/xilinx/xrt/setup.sh ;\
+                            ./{}'.format(config['EXECUTABLE_NAME'])]
             else:
-                exe_cmd = ['sh', '-c', 'cur=`pwd`; cd $AWS_FPGA_REPO_DIR ;\
+                exe_cmd = ['sh','-c','cur=`pwd`; cd $AWS_FPGA_REPO_DIR ;\
```
Comment (sampsyo) [inline buildbot/buildbot/worker.py]: Were these spaces deleted by mistake? :smiley:

Comment (ViviYe) [reply buildbot/buildbot/worker.py]: sorry I didn't notice this! Just fixed it! 😭

**Pull request merged**